guides/common/modules/proc_salt-importing-salt-states.adoc

```diff
@@ -0,0 +1,22 @@
+[id="salt_guide_importing_salt_states_{context}"]
+= Importing Salt States
+
+A Salt State configures parts of a host, for example a service or the installation of a package.
+You can import Salt States from your Salt Master to {Project}.
+The Salt Master configuration used above uses a Salt Environment called `base` which includes the Salt States stored in `/srv/salt/`.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure > Salt States* and click *Import from FQDN*.
+** Click *Edit* to assign Salt States to Salt Environments.
```

Comment:
These steps are optional? Why bullets?

Reply:
Removed the bullet points and yes, both steps are optional.